include/sparrow/buffer.hpp

```diff
@@ -56,6 +56,8 @@ namespace sparrow
         >;
         using pointer = typename base_type::pointer;
 
+        // Required to be used seen as a range iterator
```

Comment:
What is a "range iterator" in this case?

Is the following what you mean?
```suggestion
        // Required to be used seen as a `random_access_iterator`
```

Reply:
no I meant that `std::ranges::end(b)` where `b` is a buffer is valid.